src/date-range-picker/__tests__/date-range-picker-relative.test.tsx

```diff
@@ -198,7 +230,7 @@ describe('Date range picker', () => {
         relativeOptions: [],
         dateOnly: true,
       });
-      act(() => wrapper.findLabel().click());
+      act(() => wrapper.findTrigger().click());
```

Comment:
Why is this change needed? Also, does it affect customer tests if they use the same util?

Reply:
These two are the same but `findLabel` is deprecated.

Reply:
Bonus points for moving this to a separate PR, but I can accept the current one too

Reply:
Here is findLabel implementation:

```
findLabel(): ElementWrapper {
    return this.findTrigger();
}
```

Reply:
The PR title says
> Allow internal input to inherit form-field props

This change is unrelevant to the title. You can submit refactoring of these files in a separate PR